core/src/main/java/org/elasticsearch/index/query/BoolQueryBuilder.java

```diff
@@ -235,10 +235,10 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
         doXArrayContent("filter", filterClauses, builder, params);
         doXArrayContent("must_not", mustNotClauses, builder, params);
         doXArrayContent("should", shouldClauses, builder, params);
-        builder.field("disable_coord", disableCoord);
```

Comment:
Strange - I had the same idea seeing the code but also refrained from it due to the switch. Will look closer at actually changing it again.